### PR TITLE
(fix) O3-3828: Patient search crashes with Uncaught RangeError

### PR DESCRIPTION
**Pull request by chimanwadike**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [x] My work includes tests or is validated by existing tests.

## Summary
This PR fixes uncaught errors related to search results that include patients with missing dates of birth

## Screenshots

Before
image

After
image

## Related Issue

https://issues.openmrs.org/browse/O3-3828

### CODE REVIEW

**Comment by denniskigen** [comment]
In which cases would a patient not have a date of birth (either actual or estimated)?

**Comment by ibacher** [comment]
Birthdate is not technically a required field, though I think that's more a result of not every Person being a Patient.

**Review by denniskigen — APPROVED** [review]
LGTM. Thanks

File: packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.component.tsx [hunk 1]
```diff
@@ -126,7 +126,8 @@ const CompactPatientBanner = forwardRef<HTMLDivElement, CompactPatientBannerProp
                 />
               </div>
               <div className={styles.demographics}>
-                {getGender(patient.gender)} <span className={styles.middot}>&middot;</span> {age(patient.birthDate)}
+                {getGender(patient.gender)} <span className={styles.middot}>&middot;</span>{' '}
+                {patient?.birthDate && age(patient.birthDate)}
```
Comment (brandones) [inline packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.component.tsx:130]:
```suggestion
                {patient.birthDate && age(patient.birthDate)}
```

**Pull request merged**